packages/wxt/src/core/resolve-config.ts

```diff
@@ -479,14 +481,18 @@ export async function mergeBuilderConfig(
 }
 
 export async function resolveWxtUserModules(
+  rootDir: string,
   modulesDir: string,
   modules: string[] = [],
 ): Promise<WxtModuleWithMetadata<any>[]> {
+  const nodeRequire = createRequire(path.join(rootDir, 'index.js'));
+
   // Resolve node_modules modules
   const npmModules = await Promise.all<WxtModuleWithMetadata<any>>(
     modules.map(async (moduleId) => {
+      const resolvedModuleId = nodeRequire.resolve(moduleId);
```

Comment:
OK, so the main change here is that we need to resolve the ID from the root directory before importing it? That fixes the issue when WXT is symlinked?